src/docs/development/frameworks-mobile/react-native.mdx

```diff
@@ -4,51 +4,62 @@ import { Divider } from '../../../app/components';
 
 The following instructions will guide you through setting up your computer for React Native development and starting a new PX Blue project. Alternatively, you can start your project by downloading one of our code samples from [GitHub](https://github.com/pxblue) (the Login example is a good project to start from) once you configure your environment.
 
-## Setting up your environment
+<Divider />
+
+# Setting Up Your Environment
 
 You'll need to start by following the general instructions for setting up your development environment for mobile found in our [Environment Guide](/development/environment). In addition to the base requirements, you may also want the following:
 
 -   [Expo Client](https://expo.io/learn) (project creation / setup tool for React Native apps)(optional, for Expo projects)
 
-## Creating a project
+# Creating Your Application
```

Comment:
Keep consistent, use dividers in front of level one headlines

Reply:
Meant to do that and forgot...it's there now